### ISSUE TITLE
Progress Track

### ISSUE THREAD

**Issue by tirthasheshpatel** [Shrey-Viradiya/Pneumonia_Detection#1]
This is keep a track of the progress:

- [x] **Finalize the model** : Currently it is pretrained googlenet. Other options are pretrained resnet50, pretrained resnet100, or pretrained inceptionnet.
- [x] **_Data Preprocesing_** : `data.py` does some data cleaning but we may need to preprocess it too (**IFF WE HAVE TIME**)
- [x] **Training the Model** :
  - [x] Checkpoint at 90% (test set) accuracy
  - [x] Checkpoint at (90+2)% (test set) accuracy
  - [x] Checkpoint at (90+5)% (test set) accuracy
  - [ ] Checkpoint at (90+7)% (test set) accuracy

Any other tasks we need to track??
